py2verify/test_open.py

```diff
@@ -0,0 +1,24 @@
+import os
+import tempfile
+
+
+def test_open_without_import():
+    """Confirm using open without importing from io works in Python 2.7."""
+    temp_file_handle, temp_path = tempfile.mkstemp(suffix='txt', prefix='')
+    os.close(temp_file_handle)
```

Comment:
Instead of using `tempfile` directly we can rely on [pytest's fixtures for temp file creation](https://doc.pytest.org/en/latest/tmpdir.html). You can find an example of it in [`test_arg_parsing.py`](https://github.com/ryanwersal/crosswind/blob/d9e550f2bfe3ebe9803918802b1055fd98d0b8dc/crosswind/tests/test_arg_parsing.py#L9).